src/Integration.Vsix.UnitTests/Commands/HelpMenu/AboutDialogTests.cs

```diff
@@ -7,7 +27,7 @@
 namespace SonarLint.VisualStudio.Integration.UnitTests.Commands.HelpMenu
 {
     [TestClass]
-    internal class AboutDialogTests
+    public class AboutDialogTests
```

Comment:
Minor unrelated change, I noticed that there was no copyright + the class was internal (so resharper's unit tests runner doesn't recognize it)